### PR TITLE
Soyjak plushie

### PR DESCRIPTION
**Pull request by PowerfulAtom111**
## About the PR 

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added a soyjak plushie at the request of Rane, like, a month ago.

crafting_proof
plushiecrate_and_entitypanel_proof

### CODE REVIEW

File: Resources/Prototypes/Recipes/Crafting/Graphs/improvised/soyplush.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- type: constructionGraph
```
Comment (Elijahrane) [inline Resources/Prototypes/Recipes/Crafting/Graphs/improvised/soyplush.yml]: I don't think it should be constructible, it kind of bloats the construction menu a bit too much and none of the other plushies work like that

**Review by Elijahrane — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Crafting/Graphs/improvised/soyplush.yml (hunk 1 above)
Comment (Vordenburg) [reply Resources/Prototypes/Recipes/Crafting/Graphs/improvised/soyplush.yml]: I'm in agreement. Keep it out of construction.

**Comment by PowerfulAtom111** [comment]
Removed all the soyjak crafting parts of the PR, though there is one straggler file because I can't delete it from my phone.

File: Resources/Prototypes/Catalog/Fills/Crates/fun.yml [hunk 2]
```diff
@@ -32,7 +32,8 @@
         amount: 1
       - id: PlushieAtmosian
         amount: 1
-
+      - id: PlushieSoyjak
+        amount: 1
```
Comment (Vordenburg) [inline Resources/Prototypes/Catalog/Fills/Crates/fun.yml:36]: This should be added to some plushie spawners and/or mail, if it's meant to show up in more places than just the plushie crate orderable from cargo.

File: Resources/Textures/Objects/Fun/soyjak_plushie.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "y": 32
+  },
+  "states": [
+    {
+        "name": "soyjak_plushie"
+    },
+    {
+        "name": "icon"
+    }
+]
```
Comment (Vordenburg) [inline Resources/Textures/Objects/Fun/soyjak_plushie.rsi/meta.json]:
```suggestion
  ]
```

**Review by Vordenburg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**